toxcore/tox.h

```diff
@@ -2682,7 +2683,7 @@ const char *tox_err_conference_peer_query_to_string(Tox_Err_Conference_Peer_Quer
  * peer_number for the functions querying these peers. Return value is
  * unspecified on failure.
  */
-Tox_Conference_Peer_Number tox_conference_peer_count(
+uint32_t tox_conference_peer_count(
     const Tox *tox, Tox_Conference_Number conference_number, Tox_Err_Conference_Peer_Query *error);
```

Comment:
That's an interesting function. Thinking ahead, I wonder what would be the best way to change this to use opaque pointers in v0.3.0. Change it to return an array of `Tox_Conference_Offline_Peer*` opaque pointers? Or perhaps keep it as it is but add a helper function that uses this peer count as an index and returns an opaque `Tox_Conference_Offline_Peer*` to be used with other functions?

Reply:
Probably the latter, yes. Returning arrays of pointers makes things complicated and less safe (no bounds checks on C arrays).